_posts/2015-12-23-majordomo.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title:  "Implementing Majordomo"
+author: "Phil Trimble"
+date:   2015-12-23 12:00:00
+categories: technology, majordomo
+---
+
+Like most technology outfits we have work that we would like to perform 'off-cycle'. This just means that we don't want to do the work during a time that the customer is waiting. This is a very normal problem and there are many solutions to it out there in the world, so many in fact that it would take forever to list them all.
```

Comment:
>This just means that we don't want to do the work during a time that the customer is waiting
maybe
We don't want to make the customers wait for a particular task that is not critical for their experience at that moment.. something like that..